File: SUMBIG/SUMBIG.py
a = input().strip()
b = input().strip()

i = len(a) - 1
j = len(b) - 1
carry = 0
res = []

while i >= 0 or j >= 0 or carry:
    x = int(a[i]) if i >= 0 else 0
    y = int(b[j]) if j >= 0 else 0

    s = x + y + carry
    res.append(str(s % 10))
    carry = s // 10

    i -= 1
    j -= 1

print("".join(reversed(res)))

File: SUMBIG/SUMBIG.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    string a, b;
    cin >> a >> b;
    int i = a.length() - 1;
    int j = b.length() - 1;
    int carry = 0;
    string res = "";
    while (i >= 0 || j >= 0 || carry) {
        int x = (i >= 0) ? (a[i] - '0') : 0;
        int y = (j >= 0) ? (b[j] - '0') : 0;
        int s = x + y + carry;
        res += (char)(s % 10 + '0');
        carry = s / 10;
        i--;
        j--;
    }
    reverse(res.begin(), res.end());
    cout << res << endl;
    return 0;
}
